### PR TITLE
New folders, new images, new markdown, new design assets

### PR DESCRIPTION
**Pull request by cwlock**
Add new folders
Add new images (these appear in markdown files and code sample)
Add new markdown files
Add Illustrator files and Sketch file

### CODE REVIEW

File: src/docs/carousel.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Carousel
 
+## Content Disclaimer
+This UX pattern has been renamed from **Paging Panel**.
```
Comment (Reezaali) [inline src/docs/carousel.md]: Can you change it to the following: This UX design pattern was previously called Paging Panel.

File: src/docs/carousel.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Carousel takes users through a series of features or information pages before they start using the add-in.
+
+#### Code sample
+* [Carousel code sample](../templates/first-run/carousel)
+
+***
+
+![First Run - Carousel - Flowchart](../assets/markdown-images/carousel_flow.png)
+Recommended screen flow for when using a single identity provider in your add-in.
```
Comment (Reezaali) [inline src/docs/carousel.md]: Why is a single identity provider important to using the Carousel?

File: src/docs/carousel.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#### Code sample
+* [Carousel code sample](../templates/first-run/carousel)
+
+***
+
+![First Run - Carousel - Flowchart](../assets/markdown-images/carousel_flow.png)
+Recommended screen flow for when using a single identity provider in your add-in. 
+
+![First Run - Carousel - Specifications for desktop task pane](../assets/markdown-images/carousel_taskPaneCallouts.png)
+1. Carousel - Allow users to advance or skip the beginning pages of the carousel flow.
```
Comment (Reezaali) [inline src/docs/carousel.md]: Not sure what the numbers are being used for. Can you clarify? 

We don't need the "Carousel - " at the start of 1 and 2. Can you remove please?

**Comment by cwlock** [comment]
Requested changes have been made.

**Pull request merged**